index.html

```diff
@@ -0,0 +1,78 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="./style.css" />
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="styles.css" />
+  </head>
+  <body>
+    <div class="body-wrapper">
+      <!-- nav section -->
+      <div class="mobile-nav">
+        <h1>Welcome</h1>
+        <img src="./assets/main-images/bars.png" alt="" />
```

Comment:
- [ ] I can see that you have implemented the hamburger icon 😎 . However, since the icon is a trigger to navigate through the portfolio, I suggest that you consider adding this to `nav` tag to increase the portfolio accessibility to screen readers and also ensure that we are following Semantic HTML guidelines ☺️ .
- [ ] The required alt attribute specifies an alternate text for an image if the image cannot be displayed. The alt attribute provides alternative information for an image if a user for some reason cannot view it (because of a slow connection, an error in the src attribute, or if the user uses a screen reader) ☺️ . I suggest that you provide a descriptive `alt text` 👍 .